app/utils.py

```diff
@@ -296,6 +296,17 @@ def validate_image(bytestring):
         return False
 
 
+def validate_django_image(uploaded_file):
```

Comment:
add documentation for what this method returns?

Reply:
Done.